packages/salesforcedx-utils-vscode/src/telemetry/cliConfiguration.ts

```diff
@@ -5,31 +5,12 @@
  * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
  */
 
-import { which } from 'shelljs';
-import { GlobalCliEnvironment } from './commandExecutor';
+import { GlobalCliEnvironment } from '../cli';
 import { ConfigUtil } from './configUtil';
 
 export const ENV_SFDX_DISABLE_TELEMETRY = 'SFDX_DISABLE_TELEMETRY';
-export const SFDX_CLI_DOWNLOAD_LINK =
-  'https://developer.salesforce.com/tools/sfdxcli';
 export const SFDX_CONFIG_DISABLE_TELEMETRY = 'disableTelemetry';
 
-export function isCLIInstalled(): boolean {
```

Comment:
This code was not being used right now so I've removed it, we might add it in the future if needed.